libraries/app/api.cpp

```diff
@@ -302,26 +302,30 @@ namespace graphene { namespace app {
        const auto& db = *_app.chain_database();
        FC_ASSERT( limit <= 100 );
        vector<operation_history_object> result;
-       const auto& stats = account(db).statistics(db);
-       if( stats.most_recent_op == account_transaction_history_id_type() ) return result;
-       const account_transaction_history_object* node = &stats.most_recent_op(db);
+       account_statistics_object stats;
```

Comment:
Perhaps better change this to `optional<account_statistics_object>` to avoid unnecessarily object construction.

Reply:
did edit 2 and also added optional to the objects.